src/components/views/Experiment.js

```diff
@@ -126,22 +127,36 @@ export default class extends React.Component {
             <div>
                 <article id="experiment">
                     <h2>{this.props.name || this.props.slug}</h2>
-                    <section id="experiment-details">
-                        <h3>Details</h3>
-                        {maybeDescription}
-                        <section id="experiment-counts">
-                            <h4>Counts</h4>
-                            <section id="experiment-client-counts">
-                                <h5>Clients</h5>
-                                {this.getCountDL(this.props.populations, 'total_clients')}
-                            </section>
-                            <section id="experiment-ping-counts">
-                                <h5>Pings</h5>
-                                {this.getCountDL(this.props.populations, 'total_pings')}
+                    <div>
```

Comment:
Why the extra div?

Reply:
yea good call, it's an artifact of an earlier iteration